core/src/main/scala/com/karumi/shot/screenshots/ScreenshotsDiffGenerator.scala

```diff
@@ -32,11 +32,11 @@ class ScreenshotsDiffGenerator(base64Encoder: Base64Encoder) {
     val screenshot        = error.screenshot
     val originalImagePath = screenshot.recordedScreenshotPath
     val newImagePath      = screenshot.temporalScreenshotPath
-    val originalImage     = Image.fromFile(new File(originalImagePath))
-    val newImage          = Image.fromFile(new File(newImagePath))
+    val originalImage     = ImmutableImage.loader().fromFile(new File(originalImagePath))
+    val newImage          = ImmutableImage.loader().fromFile(new File(newImagePath))
```

Comment:
I was able to solve the `java.lang.IllegalStateException: Input image must store pixels as ints. Convert your input image using 0` when generating a diff with the following:
```suggestion
    val originalImage     = ImmutableImage.loader().fromFile(new File(originalImagePath)).copy(BufferedImage.TYPE_INT_ARGB)
    val newImage          = ImmutableImage.loader().fromFile(new File(newImagePath)).copy(BufferedImage.TYPE_INT_ARGB)

```

Reply:
@Jawnnypoo could you have a look at this proposed fix?